cypress/integration/providers_spec.js

```diff
@@ -0,0 +1,91 @@
+describe('Dashboard Providers Page', () => {
+  const host = process.env.DASHBOARD_HOST || 'http://localhost:3000/';
+
+  describe('When not logged in', () => {
+    it('should redirect to login page', () => {
+      cy.visit(`${host}#/providers`);
```

Comment:
If we can get #497 merged in, you won't have to preprend the host to each `visit` call. So this will just become `cy.visit('#/providers')`